core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -151,45 +151,43 @@ var TopicEditorPage = function() {
   };
 
   this.publishTopic = async function() {
-    await publishTopicButton.click();
+    await action.click('Publish Top Button', publishTopicButton);
     await waitFor.invisibilityOf(
       publishTopicButton, 'Topic is taking too long to publish.');
   };
 
+
   this.expectNumberOfQuestionsForSkillWithDescriptionToBe = async function(
       count, skillDescription) {
-    await waitFor.elementToBeClickable(
-      selectSkillDropdown, 'Skill select dropdown takes too long to appear.');
-    await selectSkillDropdown.click();
-    await element(by.css('option[label="' + skillDescription + '"]')).click();
+    await action.click('Select Skill Dropdown', selectSkillDropdown);
+    var skillDescription = element(by.css(
+      'option[label="' + skillDescription + '"]'));
+    await action.click('skill Description', skillDescription);
     await waitFor.visibilityOf(
       questionItems.first(), 'Question takes too long to appear');
     expect(await questionItems.count()).toEqual(count);
   };
 
   this.saveQuestion = async function() {
     await general.scrollToTop();
-    await saveQuestionButton.click();
+    await action.click('save QuestionButton', saveQuestionButton);
```

Comment:
```suggestion
    await action.click('Save Question Button', saveQuestionButton);
```

Please try to make these descriptions (first argument) well-formatted (start with a capital, spaces between words, correctly spelled, etc.) to keep the error messages easy to read